src/layouts/layout-container.hbs

```diff
@@ -11,6 +11,9 @@ layout: layout-base
         {{ else }}
           {{# withNewline }}{{ body }}{{/ withNewline }}
         {{/ inherit }}
+        {{# inherit stache.config.comments comments }}
```

Comment:
It's a preexisting variable. We use it in the blog pattern to enable comments there. It seemed silly to have two variables to enable comments in different places, so I just used that one.